src/clients/ClientSettings.tsx

```diff
@@ -56,17 +56,17 @@ export const ClientSettings = ({
   );
 
   const sections = useMemo(() => {
-    let result = ["generalSettings"];
+    let result = ["generalSettings", "accessSettings"];
 
     if (protocol === "saml") {
       result = [...result, "samlCapabilityConfig", "signatureAndEncryption"];
     } else if (!client.bearerOnly) {
       result = [...result, "capabilityConfig"];
     } else {
-      return [...result, "accessSettings"];
+      return [result];
```

Comment:
This would return a nested array, perhaps you meant to do:
```suggestion
      return result;
```

Reply:
Yep, my bad - thanks